runtime/onert/core/include/ir/OperandIndexSequence.h

```diff
@@ -27,6 +27,12 @@ namespace onert
 namespace ir
 {
 
+enum class Remove
+{
+  duplicated = 0x1,
+  optional = 0x10
```

Comment:
I woud like to ask that `optional` is `0x10`, why not `0x02`?

And are you going to make it bitwise enum?

Reply:
@wateret Wow. `0x10` should be `0x2`. At first, I thought running two filters (duplicated and optional) at once. However, I will run filter one by one at this time.

Also, I will rename `optional` to `undefined` in captial.